docs/INSTALL.md

```diff
@@ -172,66 +172,71 @@ In this case the filesystem will be mounted at startup, and the sfs process will
 Sync daemon component
 ===============
 
-Configuring php-sync on the central node
+Prerequisites
+--------------------
+Install php, if not already present.
+
+Make sure you have installed php together with:
```

Comment:
looks like the current situation:
https://packages.debian.org/en/wheezy/php5-cli
but this may change with newer releases, php states this as extensions which are optional, so its to the maintainer to bundle all, or make these extra packages:
http://php.net/manual/en/book.sem.php

I changed the documentaion which should make it clearer.